sdk/purview/Microsoft.Azure.Management.Purview/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("Microsoft Azure Purview Management Library")]
+[assembly: AssemblyDescription("Provides Microsft Azure Purview management functions for managing the Microsoft Azure Purview service.")]
+
+[assembly: AssemblyVersion("1.0.1.0")]
```

Comment:
1.0.0.1

Reply:
I plan to update AssemblyFileVersion as well, correct?

---
In reply to: [573375667](https://github.com/Azure/azure-sdk-for-net/pull/18237#discussion_r573375667) [](ancestors = 573375667)